yt/frontends/boxlib/fields.py

```diff
@@ -55,19 +55,19 @@ def _temperature(field, data):
 class WarpXFieldInfo(FieldInfoContainer):
 
     known_other_fields = (
-        ("Bx", ("T", ["magnetic_field_x"], None)),
-        ("By", ("T", ["magnetic_field_y"], None)),
-        ("Bz", ("T", ["magnetic_field_z"], None)),
-        ("Ex", ("V/m", ["electric_field_x"], None)),
-        ("Ey", ("V/m", ["electric_field_y"], None)),
-        ("Ez", ("V/m", ["electric_field_z"], None)),
-        ("jx", ("A", ["current_x"], None)),
-        ("jy", ("A", ["current_y"], None)),
-        ("jz", ("A", ["current_z"], None)),
+        ("Bx", ("T",   ["magnetic_field_x", "B_x"], None)),
+        ("By", ("T",   ["magnetic_field_y", "B_y"], None)),
+        ("Bz", ("T",   ["magnetic_field_z", "B_z"], None)),
+        ("Ex", ("V/m", ["electric_field_x", "E_x"], None)),
+        ("Ey", ("V/m", ["electric_field_y", "E_y"], None)),
+        ("Ez", ("V/m", ["electric_field_z". "E_z"], None)),
```

Comment:
need a comma here instead of a period

Reply:
After fixing that (no idea offhand why we weren't getting syntax errors?) I'm now getting "no old answer available" errors. So I think you just need to recreate the answers after fixing that typo and you should be good to go.